### ISSUE TITLE
[PHP] Move caret inside on unfolding code

### ISSUE THREAD

**Issue by NReib** [apache/netbeans#7713]
### Apache NetBeans version

Apache NetBeans 22

### What happened

I use code folding on arrays. Oftentimes upon unfolding the code by double clicking on it, after a short period of time the code refolds. This can be quite annoying.
An easy solution might be if after unfolding the code, the caret position would be moved inside the unfolded code.

### How to reproduce

[...] // Double click on folded array

{Caret Position}["This ist the array content"] // afterwards the code tends to fold again

[{Caret Position}"This ist the array content"] // this might prevent unwanted code folding

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

Mageia 8 (Linux)

### JDK

javac 11.0.17

### Apache NetBeans packaging

Apache NetBeans binary zip

### Are you willing to submit a pull request?

No

**Comment by junichi11** [comment]
Unfortunately, I can't reproduce it.

```
Product Version: Apache NetBeans IDE 22
Java: 19.0.2; OpenJDK 64-Bit Server VM 19.0.2+7-Ubuntu-0ubuntu322.04
Runtime: OpenJDK Runtime Environment 19.0.2+7-Ubuntu-0ubuntu322.04
System: Linux version 6.8.0-40-generic running on amd64; UTF-8; 
```

**Comment by NReib** [comment]
Indeed, also for me this is sometimes hard to reproduce - sometimes it refolds frequently, and sometimes it will show the expected behaviour and stays unfolded.

**Comment by junichi11** [comment]
Please try to use another JDK. NetBeans 22 requires JDK 17+.

**Comment by NReib** [comment]
I am using 
javac --version 
javac 17.0.11

The code sometimes stilll unfolds
